bears/general/InvalidLinkBear.py

```diff
@@ -43,8 +43,23 @@ def get_status_code(url, timeout):
     def find_links_in_file(file, timeout, link_ignore_regex):
         link_ignore_regex = re.compile(link_ignore_regex)
         regex = re.compile(
-            r'(https?://[^.:%\s_/?#[\]@\\]+\.(?:[^\s()%\'"`<>|\\]+|'
-            r'\([^\s()%\'"`<>|\\]*\))*)(?<!\.)(?<!,)')
+            r"""
+            (https?://                  # http:// or https:// as only these
+                                        # 2 are supported by the ``requests``
+                                        # library
+            [^.:%\s_/?#[\]@\\]+         # Initial Part of Domain
+            \.                          # A required dot (.)
+            (
+                (?:[^\s()%\'"`<>|\\]+)  # Domain / Subdomain Name
+            |                           # OR
+                \([^\s()%\'"`<>|\\]*\)  # Subdomain Name contained within ()
+                                        # An example can be:
+                                        # http://wik.org/Hello_(Adele_song)/200
```

Comment:
Okay :/ What should I write them as then ? They are essentially the parts after the domain. Can I call them as ``parameters`` or something ?